opencv.html.markdown

```diff
@@ -0,0 +1,141 @@
+---
+category: tool
+tool: OpenCV
+filename: learnopencv.py
+contributors:
+    - ["Yogesh Ojha", "http://github.com/yogeshojha"]
+---
+### Opencv
+
+OpenCV (Open Source Computer Vision) is a library of programming functions mainly aimed at real-time computer vision. 
+Originally developed by Intel, it was later supported by Willow Garage then Itseez (which was later acquired by Intel). 
+Opencv currently supports wide variety of languages like, C++, Python, Java etc
+
+#### Installation
+Please refer to these articles for installation of OpenCV on your computer.
+* Windows Installation Instructions: [https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutorials/py_setup/py_setup_in_windows/py_setup_in_windows.html#install-opencv-python-in-windows]
+* Mac Installation Instructions (High Sierra): [https://medium.com/@nuwanprabhath/installing-opencv-in-macos-high-sierra-for-python-3-89c79f0a246a]
+* Linux Installation Instructions (Ubuntu 18.04): [https://www.pyimagesearch.com/2018/05/28/ubuntu-18-04-how-to-install-opencv]
+### Here we will be focusing on python implementation of OpenCV
```

Comment:
Insert a newline before this line ie.

```
* Linux Installation Instructions (Ubuntu 18.04): [https://www.pyimagesearch.com/2018/05/28/ubuntu-18-04-how-to-install-opencv]()

### Here we will be focusing on python implementation of OpenCV
```

Reply:
@yogeshojha This one is still left. Please fix this as it is not rendering correctly.